larpandoradlcontent/LArDeepLearning/DeepLearningTrackShowerIdAlgorithm.h

```diff
@@ -0,0 +1,50 @@
+/**
+ *  @file   larpandoradlcontent/LArMonitoring/DeepLearningTrackShowerIdAlgorithm.h
+ *
+ *  @brief  Header file for the deep learning track shower id algorithm.
+ *
+ *  $Log: $
+ */
+#ifndef LAR_DEEP_LEARNING_TRACK_SHOWER_ID_ALGORITHM_H
+#define LAR_DEEP_LEARNING_TRACK_SHOWER_ID_ALGORITHM_H 1
+
+#include "Pandora/Algorithm.h"
+
+namespace lar_content
+{
+
+/**
+ *  @brief  DeepLearningTrackShowerIdAlgorithm class
+ */
+class DeepLearningTrackShowerIdAlgorithm: public pandora::Algorithm
+{
+public:
+    /**
+     *  @brief  Default constructor
+     */
+    DeepLearningTrackShowerIdAlgorithm();
+    virtual ~DeepLearningTrackShowerIdAlgorithm();
+
+private:
+    typedef std::map<const pandora::CaloHit*, std::tuple<int, int, int, int>> CaloHitToPixelMap;
+    typedef std::map<int, int> PixelToTileMap;
+
+    pandora::StatusCode Run();
+    pandora::StatusCode Train();
```

Comment:
Some doxygen info to add, for member functions and one member variable remaining.